test/functional/Valhalla/build.xml

```diff
@@ -56,7 +56,7 @@
 			<src path="${src}"/>
 			<src path="${transformerListener}" />
 			<compilerarg line='--add-exports java.base/jdk.internal.misc=ALL-UNNAMED --add-exports java.base/jdk.internal.value=ALL-UNNAMED' />
-			<compilerarg line='-source 20 -XDenablePrimitiveClasses' />
+			<compilerarg line='-source 21 -XDenablePrimitiveClasses' />
```

Comment:
Jdknext is Java 22 now. We should try using 22 here.

Reply:
Fixed in c2a259d